src/gc/gchandletable.cpp

```diff
@@ -22,3 +22,8 @@ void GCHandleTable::Shutdown()
 {
     Ref_Shutdown();
 }
+
+Object* GCHandleTable::ObjectFromHandle(OBJECTHANDLE handle)
+{
+    return OBJECTREFToObject(::HndFetchHandle(handle));
```

Comment:
OK. I was already planning to expose the AppDomain index for a handle table via the interface, so I'll do what @jkotas suggested above.